### ISSUE TITLE
[modal] Semantics within the modal 

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#191]
URL: https://openreferraluk.org/dashboard

`<p>` should be included for the content within the modal. Also include aria attributes to the modal as shown in the example below:

```
<div class="modal-background ">
  <div class="modal" role="dialog" aria-labelledby="dialog1Content">
    <button class="close-button button-link">×</button>
    <p id="dialog1Content">Services for Brisol and South Gloucestershire service finders</p>
  </div>
</div>
```

**Comment by kingdomlevel** [comment]
Dashboard fix - depends on #209

**Comment by kingdomlevel** [comment]
PR #209 has now been merged - removing the 'blocked' label and moving to 'To Do'

**Comment by UsmanAfzal** [comment]
Checked and approved
